### PR TITLE
Disable Oauth token handling from public java scripts 

### PR DESCRIPTION
**Pull request by Kamidu**

### CODE REVIEW

File: modules/apps/jaggery/mdm/src/mdm/modules/backend-service-invoker.js [hunk 1]
```diff
@@ -36,22 +38,34 @@ var backendServiceInvoker = function () {
...
+                return execute();
+            }
+            else {
+                return errorCallback(parse(xmlHttpRequest.responseText));
+            }
+        };
+        var accessToken = session.get(constants.ACCESS_TOKEN_PAIR_IDENTIFIER).accessToken.trim();
+        if (accessToken){
+            return execute();
+        }else {
```
Comment (prabathabey) [inline modules/apps/jaggery/mdm/src/mdm/modules/backend-service-invoker.js]: Fix formatting issues.

Comment (Kamidu) [inline modules/apps/jaggery/mdm/src/mdm/modules/backend-service-invoker.js]: These formatting issues has been fixed in a recent PR

**Comment by dakshika** [comment]
Six swap files committed with this PR

**Comment by Kamidu** [comment]
This committed swap files was removed in a recent PR.

**Pull request merged**